DetectorDescription/DDCMS/plugins/dd4hep/DDDefinitions2Objects.cc

```diff
@@ -2054,7 +2163,7 @@ void Converter<print_xml_doc>::operator()(xml_h element) const {
 static long load_dddefinition(Detector& det, xml_h element) {
   xml_elt_t dddef(element);
   if (dddef) {
-    cms::DDParsingContext context(det);
+    cms::DDParsingContext& context = *det.extension<DDParsingContext>();
```

Comment:
@cvuosalo - please, revert.

Reply:
`OutputDD4hepToDDL.cc` needs the parsing context. This approach allows the parsing context to be passed to `OutputDD4hepToDDL.cc`. At the moment, I don't see a good alternative that can be implemented on a reasonable timescale.